internal/notify/notifications.lua

```diff
@@ -303,8 +303,35 @@ local function get_bar(get_fn, get_max_fn, text, color)
     return nil
 end
 
+local function getSaveAlert()
+    local minsSinceSave = dfhack.persistent.getUnsavedSeconds()//60
+    local pen = COLOR_LIGHTCYAN
+    local savetime = 15
```

Comment:
this should get a more descriptive name (e.g. `save_time_threshold_mins`) and live outside of the function (since it's a constant)